### ISSUE TITLE
incorrect lr adjust

### ISSUE THREAD

**Issue by Zyun-Y** [Ugness/PiCANet-Implementation#28]
hi,

I found an error in your code. 

in the training part, you wrote 'if i + epo * len(dataloader) % decay_step == 0 and i != 0:'
by doing this, the learning rate never update.

the correct version I think should be 'if (i + epo * len(dataloader)) % decay_step == 0 and i != 0':

**Comment by Ugness** [comment]
I think you are right. I updated train.py.
My code has a lot of out-dated coding style, and I recommend you use pytorch's lr-scheduler.
I have a plan to update this repository with torch >= 1.4.0, but it would take some time.
Thank you for making a comment.
